cvmfs/server/cvmfs_server_common.sh

```diff
@@ -503,21 +503,15 @@ is_due_auto_garbage_collection() {
 # download a given file from the backend storage
 # @param name  the name of the repository to download from
 # @param url   the url to download from
-# @param noproxy  (optional)
 get_item() {
   local name="$1"
   local url="$2"
-  local noproxy="$3"
 
   load_repo_config $name
 
-  if [ x"$noproxy" != x"" ]; then
-    unset http_proxy && curl -f $(get_x509_cert_settings) \
-      $(get_follow_http_redirects_flag) "$url" 2>/dev/null | tr -d '\0'
-  else
-    curl -f $(get_x509_cert_settings) $(get_follow_http_redirects_flag) \
-      "$url" 2>/dev/null | tr -d '\0'
-  fi
+  curl -f -H "Cache-Control: no-cache" $(get_curl_proxy) \
```

Comment:
I recommend using `Cache-Control: max-age=0` instead of `no-cache`.  They're close, but not exactly the same thing.  With `max-age=0` a proxy may use a cached copy if it has been proven to be valid with `If-Modified-Since`, where `no-cache` means it absolutely cannot reuse a cached copy.  `max-age=0` is gentler on caches.  See the user agent section of this [stackover flow answer](https://stackoverflow.com/a/1383359/10457761) for more details.

Reply:
Sure, that's an easy change and doesn't materially impact the code.  I'll update it now.

Reply:
Updated to use `max-age=0` instead